Evil.java

```diff
@@ -48,5 +50,17 @@ public boolean isDead(){
 	public void setDead(boolean death){
 		dead = death;
 	}
+	public int upOrDown(){
+		int rand = (int) (Math.random() * 3);
+		int var=0;
+		if (rand==1)
```

Comment:
Best practice to have braces on even the simplest if statements as it's clearer what the logic flow is and prevents silly mistakes